### DISCUSSION TITLE
How to move nodes or select all next nodes and move it

### DISCUSSION THREAD

**Discussion by joacub** [ueberdosis/tiptap#2025] [General Questions]
there is no way to move nodes to other place ?
how we can move a list of nodes to other place , or how we can select all next nodes and move it to other new node ?

**Comment by cadars** [comment]
See https://tiptap.dev/guide/node-views/examples#drag-handles

**Comment by cadars** [comment]
Also of note: https://tiptap.dev/guide/custom-extensions#:~:text=The%20schema%20even%20allows%20to%20make%20your%20nodes%20draggable
